pymatgen/core/surface.py

```diff
@@ -1231,25 +1222,25 @@ def get_slabs(
                 sorted by the number of bonds broken.
         """
 
-        def gen_possible_shifts(ftol: float) -> list[float]:
-            """Generate possible shifts by clustering z coordinates.
+        def gen_possible_terminations(ftol: float) -> list[float]:
```

Comment:
@DanielYang59 this rename looks like a breaking change? am i missing something?

Reply:
This is an internal (private) function of `get_slabs`, I don't expect this to be breaking :)

Reply:
ah, i was on my phone and missed the double indent. always better to review on a big screen :)

Reply:
All good. But I'm not sure what caused all CI runs to fail? Perhaps some issue with the latest `chgnet`?